.github/workflows/promptflow-evals-unit-test.yml

```diff
@@ -72,7 +72,11 @@ jobs:
         run: poetry run pip install -e ../promptflow-recording
         working-directory: ${{ env.WORKING_DIRECTORY }}
       - name: run unit tests
-        run: poetry run pytest -m unittest --cov=promptflow --cov-config=pyproject.toml --cov-report=term --cov-report=html --cov-report=xml --cov-fail-under=63
+        id: run_unit_tests
+        run: |
+            poetry run pytest -m unittest --cov=promptflow --cov-config=pyproject.toml --cov-report=term --cov-report=html --cov-report=xml --cov-fail-under=63
+            export cov_pct=`grep "data-ratio" htmlcov/index.html | tail -n1 | sed -r 's/.*>([0-9]+)%.*/\1/g'`
+            poetry run python ../../scripts/code_qa/report_to_app_insights.py --activity test_coverage_pct --value $cov_pct --git-hub-action-run-id ${{ github.run_id }} --git-hub-workflow ${{ github.workflow }} --git-hub-action ${{ github.action }} --git-branch ${{ github.ref }}
```

Comment:
Not sure if we need this. If we do, can we move it to the report section?

Reply:
Removed.